.github/workflows/pre-release.yaml

```diff
@@ -23,5 +23,5 @@ jobs:
       - name: Release dry run
         env:
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
-          GITHUB_TOKEN: ${{ secrets.GH_BOT_ACCESS_TOKEN }}
```

Comment:
This environment variable should be setup automatically. We don't need to set it. AFAIK github automatically exposes this env variable `GITHUB_TOKEN`